packages/shorebird_cli/lib/src/commands/account/account_usage_command.dart

```diff
@@ -0,0 +1,85 @@
+import 'dart:async';
+
+import 'package:barbecue/barbecue.dart';
+import 'package:mason_logger/mason_logger.dart';
+import 'package:shorebird_cli/src/code_push_client_wrapper.dart';
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/logger.dart';
+import 'package:shorebird_cli/src/shorebird_config_mixin.dart';
+import 'package:shorebird_cli/src/shorebird_validation_mixin.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+/// {@template account_usage_command}
+/// `shorebird account usage`
+/// Get usage information for your Shorebird account.
+/// {@endtemplate}
+class AccountUsageCommand extends ShorebirdCommand
+    with ShorebirdConfigMixin, ShorebirdValidationMixin {
+  @override
+  String get description => 'Get usage information for your Shorebird account.';
+
+  @override
+  String get name => 'usage';
+
+  @override
+  Future<int> run() async {
+    try {
+      await validatePreconditions(checkUserIsAuthenticated: true);
+    } on PreconditionFailedException catch (e) {
+      return e.exitCode.code;
+    }
+
+    final progress = logger.progress('Fetching usage');
+    final List<AppUsage>? appsUsage;
+    try {
+      appsUsage = await codePushClientWrapper.codePushClient.getUsage();
+      progress.complete();
+    } catch (error) {
+      progress.fail(error.toString());
+      return ExitCode.software.code;
```

Comment:
Accessing the codePushClient through the wrapper seems a bit wrong. I'd add a `getUsage` method to CodePushClientWrapper and move this logging and error handling there.

Reply:
yeah good call, done.